solutions/swb-reference/integration-tests/tests/isolated/costCenter/create.test.ts

```diff
@@ -86,28 +129,29 @@ describe('Cost Center negative tests', () => {
         }
       });
     });
+
     test('with invalid accountId', async () => {
       const invalidAccountIds: string[] = generateInvalidIds(resourceTypeToKey.account.toLowerCase());
-      const invailidCreateRequests = invalidAccountIds.map((accountId) => ({
+
+      const invalidCreateRequests = invalidAccountIds.map((accountId) => ({
         accountId,
         name: validCreateRequest.name,
         description: validCreateRequest.description
       }));
-      await Promise.all(
-        invailidCreateRequests.map(async (invalidCreateRequest) => {
-          try {
-            await adminSession.resources.costCenters.create(invalidCreateRequest, false);
-          } catch (error) {
-            checkHttpError(
-              error,
-              new HttpError(400, {
-                error: 'Bad Request',
-                message: `accountId: Invalid ID`
-              })
-            );
-          }
-        })
-      );
+
+      for (const invalidRequest of invalidCreateRequests) {
```

Comment:
was getting intermittent 502s when using `Promise.all`, so I switched to a for loop which appeared to be more stable.

Reply:
Lets keep it for now since its already present. I'm not sure why Fernando initially did different variations but I think changing that would be outside the scope of this update